.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     'comma-dangle': 'off',
     'implicit-arrow-linebreak': 'off',
     'import/prefer-default-export': 'off',
+    'jsx-a11y/control-has-associated-label': 'off',
```

Comment:
This rule seems unnecessary to enforce since screen reader support doesn't seem necessary.